src/system/utility.ts

```diff
@@ -29,27 +29,34 @@ export const visuallyHiddenProp = ({ visuallyHidden }: any): any => {
   }
 };
 
-export const backgroundOpacityProp = ({
-  backgroundOpacity,
-  backgroundColor,
-}: {
-  backgroundOpacity: number | string;
-  backgroundColor: keyof typeof colors;
-}): any => {
+export const backgroundOpacityProp = ({ backgroundOpacity, backgroundColor }: any): any => {
```

Comment:
Why remove the explicit typings?

Reply:
TS was failing with 3 pages of messages that were un-debuggable. Ultimately, the external typings (the ones the developers see) remain the same. I just changed the internal ones (the one internal to the typed-system) to satisfy some weird messages.

P.S. I've checked and major libraries like `chakra-ui` do the same thing cause `@styled/system` does not play great with TS